src/goMain.ts

```diff
@@ -72,6 +72,11 @@ export function activate(ctx: vscode.ExtensionContext): void {
 		testCurrentFile(goConfig['testTimeout']);
 	}));
 
+	ctx.subscriptions.push(vscode.commands.registerCommand('go.test.previous', () => {
+		let goConfig = vscode.workspace.getConfiguration('go');
+		testPrevious();
```

Comment:
Actually - looks like you are intentionally not passing `testTimeout` - so should drop the `goConfig` line.